sklearn/feature_extraction/dict_vectorizer.py

```diff
@@ -312,7 +312,9 @@ def get_feature_names(self):
         return self.feature_names_
 
     def restrict(self, support, indices=False):
-        """Restrict the features to those in support.
+        """Gives support for feature selection by restricting the features to those in support.
```

Comment:
I think it is 84. Should the docstrings be less than 79 characters? Should I add a line break? Or perhaps, try to shorten this line and add that information in the estimator line?